examples/complete/variables.tf

```diff
@@ -42,7 +42,7 @@ variable "abac_valid_tags" {
   default     = {}
 }
 
-variable "name-prefix" {
+variable "name_prefix" {
```

Comment:
run `make terraform/docs` so that this gets reflected in the README table

Reply:
Done. It will be reflected in the next commits.